backend/cms/templates/csrf_failure.html

```diff
@@ -0,0 +1,15 @@
+{% extends "_raw.html" %}
+
+{% block raw_content %}
+<div id="no-session" class="px-4 py-10">
+    <div class="w-full max-w-xs mx-auto">
+        <h1 class="mb-4">CSRF Fehler</h1>
+        <p class="mb-4">
+            Bitte versuchen Sie die Seite neu zu laden.
```

Comment:
I would vote for directly using English language in source. And we should start with translatable strings.